.github/workflows/regression.yml

```diff
@@ -0,0 +1,62 @@
+name: "Regression Tests"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to test against"
+        required: true
+        default: "latest"
+  release:
+    types: [published]
```

Comment:
Running the super-permutation of tests out-of-band was the purpose of the PR, while keeping the regular E2E in-band.

Since that's not desired, I can just close this one.

Reply:
I suspect we are misunderstanding each other 🤦

Wasn't the idea to reduce the build time for PR builds? I think that's super valuable.

But I still think we need to run the full permutation in release builds.